network/connection_wrapper_test.go

```diff
@@ -31,6 +31,25 @@ import (
 var TestClientID = []byte("client")
 var TestServerID = []byte("server")
 
+var TestCertVerifier CertVerifier
+
+func init() {
```

Comment:
why not to do it with factory method which use where it need? to not clog global scope of variables.
```
func NewTestCertVerifier(t *testing.T) CertVerifier{
ocspConfig, err := NewOCSPConfig("", "yes", "prefer")
	if err != nil {
		t.Fatal("Cannot create OCSP config")
	}

	crlConfig, err := NewCRLConfig("", "use")
	if err != nil {
		t.Fatal("Cannot create CRL config")
	}

	TestCertVerifier, err = NewCertVerifierFromConfigs(ocspConfig, crlConfig)
	if err != nil {
		t.Fatal("Cannot create certificate verifier")
	}
        return TestCertVerifier
}
```

Plus you use panics and... I don't know how test runner will handle it. Is it will recover and continue execution or will stop on first panic...

Reply:
>  don't know how test runner will handle it. Is it will recover and continue execution or will stop on first panic

That'll kill the test runner even before it starts running tests.

Well, I'd avoid using global test fixtures which can cause weird *side-effects* later with concurrent test execution.